Yosemite/Yosemite/Tools/EntityListener.swift

```diff
@@ -66,22 +68,24 @@ private extension EntityListener {
     /// Handles the ContextObjectDidChange Notification triggered by the ViewContext.
     ///
     func viewContextDidChange(notification: Notification) {
-        guard let note = ManagedObjectsDidChangeNotification(notification: notification) else {
-            return
-        }
+        queue.sync {
```

Comment:
❓ Do we also need a queue here? i.e. Do we have to execute these callbacks synchronously?